documentation/release_notes.markdown

```diff
@@ -10,6 +10,14 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.11.0
+
+Released 30 April 2020
+
+### New features
+
+- The `puppetserver ca` CLI tool has been updated to version 1.7.0. It will now show any authorization extensions that exist when listing certificates or csrs.
```

Comment:
Do we usually capitalize "CSRs"?

Reply:
Good question. It looks like there's only one other spot we mention them and it's capitalized. I'll update.